lib/plug/exceptions.ex

```diff
@@ -17,9 +20,18 @@ defprotocol Plug.Exception do
   """
   @spec status(t) :: Plug.Conn.status
   def status(exception)
+
+  @doc """
+  Receives an exception and returns a message safe to display to users.
+  """
+  @spec message(t) :: nil | String.t
+  def message(exception)
 end
 
 defimpl Plug.Exception, for: Any do
   def status(%{plug_status: status}) when is_integer(status), do: status
   def status(_), do: 500
+
+  def message(%{plug_status: status, message: message}) when is_integer(status), do: message
```

Comment:
I can remove it but plug_status becomes less useful because we can almost never use it. We need to implement the protocol to give a message.